src/groundStation/groundStation.py

```diff
@@ -320,6 +320,57 @@ def getOptions(self):
         return self.parser.parse_args(sys.argv[1:])
 
 
+class getEmbededCSPData:
+    def __init__(self, data):
+        self.data = data
+        self.opts = options()
+        self.csp = groundStation(self.opts.getOptions())
+        self.filename = input("enter file name: ")
```

Comment:
```suggestion
    def __init__(self, data, filename='schedule.txt'):
        self.data = data
        self.opts = options()
        self.csp = groundStation(self.opts.getOptions())
        self.filename = filename
```
Instead of calling `input()` here for the filename, can we move that to cli.py and instead make filename a parameter with a default value in the constructor. It makes it more convenient for external modules (like the website's comm.py) to have their own way of selecting the schedule file

Reply:
resolved using argparse